morango/sync/syncsession.py

```diff
@@ -423,25 +528,19 @@ def initiate_push(self, sync_filter):
         self._close_transfer_session()
 
     def initiate_pull(self, sync_filter):
-
-        logger.info("Initiating pull sync")
         self._create_transfer_session(False, sync_filter)
 
         if self.current_transfer_session.records_total == 0:
-            logger.info("There are no records to transfer")
             self._close_transfer_session()
             return
 
-        logger.info(
-            "{} records have been queued server side for transfer".format(
-                self.current_transfer_session.records_total
-            )
-        )
-        # pull records and close transfer session upon completion
-        logger.info("Beginning pulling of data...")
```

Comment:
I couldn't find where these specific ones were replaced by use of `OperationLogger` -- it'd be helpful to still print these for when we're running from the command line. Same for pushing.

Reply:
With the signals, the consumer of the morango API can now implement their own logging. This avoids unnecessary noise.